generators/app/index.js

```diff
@@ -147,8 +147,8 @@ function toCamelCase(aString) {
   var tokens = aString.split(" ");
 
   var camelString = "";
-  tokens.map(function(token) {
-    camelString += token[0].toUpperCase();
+  tokens.map(function(token, index) {
+    index !== 0 ? camelString += token[0].toUpperCase() : camelString += token[0];
```

Comment:
The ternary operator makes this a bit harder to read - could we break it into a normal if statement instead, and perhaps add a comment explaining why this code is needed? Thanks!

Reply:
Sure :)

Reply:
@yochannah Fixed and commented.

Reply:
This looks better! Let's just make sure it passes linting?

Reply:
It does.